include/unifex/scheduler_concepts.hpp

```diff
@@ -133,5 +170,12 @@ inline constexpr struct now_cpo {
   }
 } now;
 
+inline constexpr struct bulk_schedule_cpo {
+  template<typename BulkScheduler, typename Range>
+  auto operator()(BulkScheduler&& scheduler, Range&& range) const {
+
```

Comment:
Uh, this line intentionally left blank? :-D I'm guessing this got included by accident.

Reply:
This whole `bulk_schedule` CPO shouldn't be in this PR.
Will remove.